packages/web/src/common/store/pages/signon/sagas.js

```diff
@@ -113,6 +113,16 @@ function* getArtistsToFollow() {
   yield put(signOnActions.setUsersToFollow(users))
 }
 
+function* fetchDefaultFollowArtists() {
```

Comment:
nice, so the ideas is that these are in the cache now, so when user follows them, they will be updated appropriately?

Reply:
yes! basically just trying to fetch the users in parallel early in the flow to speed up the follow process during sign up (and thus maximizes the chance that the follows will be complete once the users hit the feed)